### ISSUE TITLE
Apache | Cannot start addon

### ISSUE THREAD

**Issue by niro1987** [FaserF/hassio-addons#2]
```text
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
SSL is deactivated
Here is your web file architecture.
ls: /var/www/localhost/htdocs/: No such file or directory
[cmd] /run.sh exited 1
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
```
System Health | --
 -- | --
arch | x86_64
chassis | vm
dev | false
docker | true
docker_version | 19.03.8
hassio | true
host_os | HassOS 4.11
installation_type | Home Assistant OS
os_name | Linux
os_version | 5.4.50
python_version | 3.8.3
supervisor | 229
timezone | Europe/Amsterdam
version | 0.113.0
virtualenv | false

**Comment by FaserF** [comment]
Until now I had no test with a VM installation. 
Can you tell me where you have put your website in the VM?

As the log file says, it can't find your Website folder.

**Comment by niro1987** [comment]
I figured the addon would create the `/htdocs` folder in `/share` with something like a generic `index.html`.
After having created the `/htdocs` folder, the addon started without error.
Perhaps the docs could be made a bit more explicit in stating that you should create the `/htdocs` folder before starting the addon or even better; the addon could create a folder by the name of the configured `website_name` and link to that.

**Comment by FaserF** [comment]
I have just released Version 1.2.0, now it should be better visable that a folder needs to be created before starting the addon.
You can now also change the path, where you will put your webfolder structure.
